chaos.py

```diff
@@ -50,6 +50,13 @@ def main():
     log.info("Setting description to {desc}".format(desc=settings.REPO_DESCRIPTION))
     github_api.repos.set_desc(api, settings.URN, settings.REPO_DESCRIPTION)
 
+    log.info("Blocking users specified in blocklist")
+    with open(join(dirname(__file__), "blocklist"), "r") as blocklist:
+        try:
+            github_api.users.set_blocklist(api, blocklist.readlines())
+        except:
+            log.exception("Failed to update blocklist")
```

Comment:
I'm into the idea of wrapping non-core functionality in good error handling :+1: